Dockerfile

```diff
@@ -1,21 +1,31 @@
 # Builder
-
 FROM golang:1.12.6-alpine3.9 as builder
 
-WORKDIR /go/src/github.com/openbank/openbank
+RUN apk --no-cache add ca-certificates git
+
+WORKDIR /src/github.com/openbank/openbank
+
+# copying go.mod and go.sum first
+# this to enable docker cache for cache all go dependencies
+# go mod download command will always download all dependencies while we make changes to the code.
+# this will only re-use the downloaded dependencies for exactly the same build.
+# Copying the go.mod and go.sum before all files, will check the dependencies from docker cache, 
+# and will download dependencies ONLY IF go.mod and go.sum change.
```

Comment:
This is far too long, and the wording is off. For example, you mix sentences that are capitalized and end with a period, with sentences that don't.

It would take too much effort to point out the issues, so here's a rewrite:

> Copy go.mod and go.sum and run 'go mod download'. This way, docker will keep this costly download step cached as long as the go.mod and go.sum files don't change.